corehq/apps/locations/tests/test_location_expressions.py

```diff
@@ -69,45 +69,71 @@ def test_location_not_found(self):
         self._check_expression(doc, None)
 
 
+def setup_module():
+    domain = "a-song-of-ice-and-fire"
+    domain_obj = create_domain(domain)
+    continent_location_type = LocationType(
+        domain=domain,
+        name="continent",
+        code="continent",
+    )
+    continent_location_type.save()
+    kingdom_location_type = LocationType(
+        domain=domain,
+        name="kingdom",
+        code="kingdom",
+        parent_type=continent_location_type,
+    )
+    kingdom_location_type.save()
+    city_location_type = LocationType(
+        domain=domain,
+        name="city",
+        code="city",
+        parent_type=kingdom_location_type,
+    )
+    city_location_type.save()
+
+    parent = SQLLocation(
```

Comment:
nit: I got a little confused looking at tests and translating parent to continent, child to kingdom etc. Could change these variable names to their location types?